docs/imagecustomizer/cli.md

```diff
@@ -31,7 +31,7 @@ The file path to write the final customized image to.
 
 The image format of the the final customized image.
 
-Options: vhd, vhd-fixed, vhdx, qcow2, raw, and iso.
+Options: vhd, vhd-fixed, vhdx, qcow2, raw, iso, and cosi.
```

Comment:
we should probably expand a bit more on what cosi is in a separate markdown file

Reply:
Will add a doc for cosi in next phase 2 pr